smoke/ospf6_frr_test.sh

```
@@ -0,0 +1,96 @@
+# Convergence takes ~40s, be patient.
+attempts=60
+while ! $(vtysh -c 'show ipv6 ospf6 neighbor json' | jq '.neighbors[] | select(.neighborId=="2.2.2.2") | .state == "Full"' -e > /dev/null) ; do
+	sleep 1
+	if [ "$attempts" -le 0 ]; then
+		fail "OSPF6 failed to connect to neighbor."
+	fi
+	attempts=$((attempts - 1))
+done
+
+attempts=20
+while ! $(vtysh -c 'show ipv6 route ospf6 json' | jq '."2001:db8:1000::/64"[0].protocol == "ospf6"' -e > /dev/null) ; do
```

Shellcheck (0.11.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)